pinot-core/src/main/java/com/linkedin/pinot/core/segment/store/SegmentLocalFSDirectory.java

```diff
@@ -38,6 +39,13 @@
 class SegmentLocalFSDirectory extends SegmentDirectory {
   private static Logger LOGGER = LoggerFactory.getLogger(SegmentLocalFSDirectory.class);
 
+  // matches most systems
+  private static final int PAGE_SIZE_BYTES = 4096;
+  // Prefetch limit...arbitrary but related to common server memory and data size profiles
+  private static final long MAX_MMAP_PREFETCH_PAGES = 100 * 1024 * 1024 * 1024 / PAGE_SIZE_BYTES;
+  private static final double PREFETCH_SLOWDOWN_PCT = 0.67;
```

Comment:
Can you add comments on how this 0.67 was derived? Also, if there's any experimental data supporting the same, please share that as well.